source/tutorials/tutorial.md

```diff
@@ -20,39 +20,41 @@ This tutorial assumes you have already installed the mod or modpack. If you'd li
 
 ## Quick Walkthrough 
 
-Before you get started on your new colony, you should find your perfect location. You need a large enough area for a full city.
+Before you get started on your new colony, you should find your perfect location. You need a large enough area for a full city - at least 8x8 chunks, and decently flat.
 
-You should also gather as many resources as you can. Once you figure out which style you want, you should start gathering enough wood for that style, as well as cobble, coal, iron, flowers, and some saplings for other trees not in your style as they are often used as decoration items.
-
-You should also gather enough food to feed yourself and your initial citizens until you can get workers who gather and cook food. 
+You should also gather as many resources as you can - wood, as well as cobble, coal, iron, flowers, string, leather, wool and some saplings. 
 
 ### Step 1: Supply Camp/Supply Ship
   
-  Once you have your location determined, you must place a [Supply Ship](../../source/items/supplycampandship) or [Supply Camp](../../source/items/supplycampandship). You can only place one of these. The Supply Camp and Ship have the Town Hall block you need to officially start your colony. They also have some other supplies you may need during the building of your colony. Once you have the Town Hall block and the [building tool](../../source/items/buildingtool), the remaining camp or ship is just decoration. You can leave it if you wish or tear it down.
+  Once you have chosen a location for your colony, you must place a [Supply Ship](../../source/items/supplycampandship) or [Supply Camp](../../source/items/supplycampandship). You can only place one of these. The Supply Camp/Ship gives you the Town Hall block you need to officially start your colony. They also have some other supplies you may need during the building of your colony. Once you have the Town Hall block and the [building tool](../../source/items/buildingtool), the remaining camp/ship is just decoration. You can leave it if you wish or tear it down.
   
-### Step 2 Town Hall
+### Step 2: Town Hall
   
-  The [Town Hall](../../source/buildings/townhall) block is what actually creates the colony. This is where you start using the building tool to place buildings. Using the building tool ensures you can see exactly where the building will be, which direction it will face, and if it is at the right y level (some buildings need to be raised to the correct y level).
+  The [Town Hall](../../source/buildings/townhall) block is what actually creates the colony, and functions as the center of it. This is where you start using the building tool to place buildings. Using the building tool ensures you can see exactly where the building will be, which direction it will face, and if it is at the right y level (some buildings need to be raised to the correct y level).
```

Comment:
```suggestion
  The [Town Hall](../../source/buildings/townhall) block is what actually creates the colony. It also functions as the center of it. This is where you start using the building tool to place buildings. Using the building tool ensures you can see exactly where the building will be, which direction it will face, and if it is at the right Y level (some buildings need to be raised to the correct Y level).
```